CONTRIBUTING.md

```diff
@@ -83,7 +83,30 @@ Flutter tests use [package:flutter_test](https://github.com/flutter/flutter/tree
  * `cd packages/flutter_tools`
  * `dart test/all.dart`
 
-`flutter test --flutter-repo` is a shortcut for those working on the flutter repository itself which runs all tests inside the `flutter` package regardless of the current working directory.
+Those working on the Flutter repo might find it useful to have a command-line alias for running Flutter's own tests
+regardless of the current working directory. For example, in your `.bashrc` on Linux, or `.profile` on Mac, define:
+
+```sh
+# Where you cloned the flutter repo
+export FLUTTER_REPO_DIR=/path/to/flutter/repository
+
+test-flutter-package() {
+  PACKAGE=${1:-"flutter"}
+  PACKAGE_DIR="$FLUTTER_REPO_DIR/packages/$PACKAGE"
+  echo "Testing $PACKAGE_DIR"
+  (cd $PACKAGE_DIR; flutter test)
+}
+```
+
+Then run it from the command-line like this:
+
+```sh
+# To test `package:flutter`
+test-flutter-package
+# To test `package:flutter_sprites`
+test-flutter-package flutter_sprites
+```
+
```

Comment:
I wouldn't bother talking about this. People should just use `test.sh` and if they know that they can look in that file to figure out how to do individual subparts of it.